nano/templates/pai_radio.tmpl

```diff
@@ -38,10 +38,10 @@ code/modules/mob/living/silicon/pai/software_modules.dm
 	<div class="itemContent">
 		{{if value.listening}}
 			{{:helper.link("On", '', {"stopic":"radio", "channel":value.name, "listen":1, "nowindow":1}, 'selected')}}
-			{{:helper.link("Off", '', {"stopic":"radio", "channel":value.name, "listen":1, "nowindow":1})}}
+			{{:helper.link("Off", '', {"stopic":"radio", "channel":value.name, "listen":0, "nowindow":1})}}
```

Comment:
Confused by this change; unless I'm looking at an outdated file the `"listen"` key induces a toggle. Providing a value of `0` either encodes as a number, in which case this button will do nothing, or as a `"0"` string, in which case this change has no effect (and I don't recall which).